.github/labeler.yml

```diff
@@ -38,3 +38,11 @@ tooling:
           prettier.config.js
           turbo.json
           vercel.json
+ci:
+  - changed-files:
+      - any-glob-to-any-file: |
+          .github/actions/**
+          .github/workflows/**
+          .github/crates-filters.yml
+          .github/external-crates-filters.yml
+          .github/labeler.yml
```

Comment:
Any reason not to just use `.github/**`?

Reply:
Some stuff in there is actually not CI

Reply:
Yeahhhhh but it's CI adjacent. Not sure I would distinguish especially since we don't have a different tag for everything else

Reply:
Alright